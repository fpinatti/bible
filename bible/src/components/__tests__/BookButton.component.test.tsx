import { render } from "@testing-library/react"
import BookButton from "../BookButton.component"
import { BrowserRouter } from "react-router-dom"

describe(BookButton, () => {
    test('test 1', () => {
        // arrange
        const wrapper = render(<BrowserRouter>
            <BookButton id={1} name="test name" />
        </BrowserRouter>)
        // act/assert
        expect(wrapper).toMatchSnapshot()
    })
})