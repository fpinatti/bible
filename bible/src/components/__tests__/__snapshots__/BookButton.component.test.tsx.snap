// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookButton test 1 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <a
        class="underline p-5 m-2 rounded bg-cyan-950 text-stone-100 hover:bg-cyan-900 hover:text-stone-50"
        href="/1"
      >
        test name
      </a>
    </div>
  </body>,
  "container": <div>
    <a
      class="underline p-5 m-2 rounded bg-cyan-950 text-stone-100 hover:bg-cyan-900 hover:text-stone-50"
      href="/1"
    >
      test name
    </a>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
